src/System.Drawing.Common/tests/System/Drawing/Drawing2D/GraphicsPathIteratorTests.cs

```
@@ -320,14 +333,14 @@ public static IEnumerable<object[]> CopyData_StartEndIndexesOutOfRange_TestData(
 
     [Theory]
     [MemberData(nameof(CopyData_StartEndIndexesOutOfRange_TestData))]
-    public void CopyData_StartEndIndexesOutOfRange_ReturnsExpected(PointF[] points, byte[] types, int startIndex, int endIndex)
+    public void CopyData_WithData_StartEndIndexesOutOfRange_ThrowsArgumentException(PointF[] points, byte[] types, int startIndex, int endIndex)
     {
         PointF[] resultPoints = new PointF[points.Length];
         byte[] resultTypes = new byte[points.Length];
 
         using GraphicsPath gp = new(points, types);
         using GraphicsPathIterator gpi = new(gp);
-        Assert.Equal(0, gpi.CopyData(ref resultPoints, ref resultTypes, startIndex, endIndex));
+        AssertExtensions.Throws<ArgumentException>(null, () => gpi.CopyData(ref resultPoints, ref resultTypes, startIndex, endIndex));
```

Do we need to update documentation since this is now throwing for these cases?
Also.. how are we getting exception for the last case where startIndex=2 -- is it from the PInvoke?

---

It is from our own check. That said, I'm going to be more explicit in the checks so it throws if start is greater than end.

I don't think we need to update the docs as we already throw for some invalid inputs, we just weren't consistent.